shared/editor/plugins/UploadPlugin.ts

```diff
@@ -76,17 +75,13 @@ export class UploadPlugin extends Plugin {
                 .then((response) => response.blob())
                 .then((blob) => {
                   const fileName = fileNameFromUrl(imageSrc) ?? "pasted-image";
-                  const ext = extension(blob.type) ?? "png";
```

Comment:
Honestly, this wasn't particularly useful logic anyway – and only hits in rare cases that the uploaded file has no extension